sway/desktop/transaction.c

```diff
@@ -284,7 +284,7 @@ static void disable_container(struct sway_container *con) {
 }
 
 static void arrange_container(struct sway_container *con,
-		int width, int height, bool title_bar, int gaps);
+		int width, int height, bool title_bar, bool lone_tab, int gaps);
```

Comment:
Why can't we set `title_bar = true` instead of adding the second parameter?

Reply:
I was confused by an extra titlebar but now I realize it was unrelated to this change. Back to the original commit.